db/payment_icons.yml

```diff
@@ -1227,13 +1227,13 @@
   label: Scalapay
   group: other
 -
+
+  name: aplazo
+  label: Aplazo
+
   name: paybylink
   label: Pay By Link
-  group: other
--
-  name: snap_checkout
-  label: Snap Checkout
-  group: other
+
```

Comment:
Something odd going on here. Can we rebase this PR to resolve this diff? Should not be removing `snap_checkout`.